twine/settings.py

```diff
@@ -14,9 +14,9 @@
 # limitations under the License.
 from __future__ import unicode_literals
 
-from twine import exceptions
 from twine import repository
 from twine import utils
+import twine.exceptions
```

Comment:
Could you share why you made this particular change? It seems unnecessary to me.

Reply:
I did this here (and everywhere else) to make it more clear that these are Twine's exceptions when looking at their usage in various places. IMO the alternative would be to rename the exceptions to something like `TwineInvalidConfigurationException`.